### PR TITLE
Translation update, Some bug fix

### PR DESCRIPTION
**Pull request by lishaoxia1985**
Some update:
- Chinese translation update.
- Scrolling World screen from up to down is limited now.
- Remove ProgressBarVertical of tech button in pick screen.
- Remove some blank key when we use Fonts.turn.

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/Nation.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Nation : INamed {
         addUniqueUnitsText(textList, ruleset)
         addUniqueImprovementsText(textList, ruleset)
 
-        return textList.joinToString("\n").tr().trim()
+        return textList.joinToString("\n").trim()
```
Comment (HadeanLake) [inline core/src/com/unciv/models/ruleset/Nation.kt:106]: Nation.kt is in PR already

**Comment by yairm210** [comment]
Why did you limit the zoom? This means that we can barely zoom now

**Comment by lishaoxia1985** [comment]
@yairm210 two reason:
1. zooming too big is meaningless.
2. when you zoom too big,  and then scrolling the world screen,  you will find that going to the edge of world screen map is impossible. so the value of zoom should be limited. You can test this and decide the value.

**Comment by yairm210** [comment]
I see  that you and hadean made a similar fix in the nations' tr(), do just resolve that and we'll meet this :)

**Comment by lishaoxia1985** [comment]
@yairm210 now it's OK.

**Comment by yairm210** [comment]
merge translations

**Pull request merged**